### DISCUSSION TITLE
[Question] - If I download a just Sayori mod, will it affect MAS?

### DISCUSSION THREAD

**Discussion by JinSakaiM** [Monika-After-Story/MonikaModDev#8731] [QA - tech support + general] [answered]
I am a sayorian my self, but I like Monika at the same time, I was fixing to download the just sayori mod, but I thought about this question, I just gotta make sure it cant affect MAS. (Edit) - Im just being a little precautious

**Comment by JinSakaiM** [comment]
And I dont wanna make another issue, so ill go ahead and ask this question as well, if I download the Blue Skies mod will it also affect MAS?

**Comment by zwartkop2** [comment]
It would be better to have it on a separate install of DDLC.

**Comment by lunulae** [comment]
They won't affect your save, but you will need to have a separate install of DDLC for each mod.

**Comment by JinSakaiM** [comment]
ok thank you for the information, do I have to like put my persistent in the newely downloaded DDLC?

**Comment by JinSakaiM** [comment]
It be better playing it without my persistent (Edit)- not better but it would work, although theres none of my data on there

**Comment by JinSakaiM** [comment]
Id still like my previous question answered though just in case

**Comment by lunulae** [comment]
You shouldn't put your MAS persistent in the new install. This is separate from your DDLC persistent, which is fine to use.

Most mods should work fine without you moving anything persistent-wise, but this may differ depending on the specific mod, so you may want to get tech support from their end if things seem to be running strangely. As always when installing new things, backing up a copy of persistent files is usually a good practice.

**Comment by JinSakaiM** [comment]
Ok, so I should backup my persistent very often?

**Comment by JinSakaiM** [comment]
I think I prefer to not use a persistent, If anything goes wrong I'll make a new issue

**Comment by JinSakaiM** [comment]
but do I need a persistent for the just Sayori mod?

**Comment by Booplicate** [comment]
You can *not* use persistent... it's being generated and used automatically. We make a few backups for you, but you should still backup it.
We don't know how other mods affect MAS since that's not our code. MAS does not interact with other mods.

**Comment by JinSakaiM** [comment, marked as answer]
Ik, I just played the mod, and I didnt need a persistent, and sorry for bringing it up out of the blue, but the answer is basically download the mod, (not sure abt this part) and ddlc
